requirements.dev.txt

```diff
@@ -1,22 +1,23 @@
-autoflake
+# packaging
```

Comment:
Why we don't use extra dependencies list `dev` in the TOML file? Why a separate txt file is needed? Same question for requirements.txt

Reply:
That's for historical reasons. It looks putting everything in `pyproject.toml` is easier to manage. I'll change it in a new commit.